src/lib/dev_mockData/nonceSigs.ts

```diff
@@ -0,0 +1,45 @@
+export type NonceSignature = {
```

Comment:
Same for these, we should use the format of the test sigs that I sent from Cameron

Reply:
Yuppers